src/core/utils/TextTruncation.test.tsx

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { render } from '@testing-library/react';
+import React from 'react';
+import TextTruncation from './TextTruncation';
+
+it('should truncate the text', () => {
+  const { container } = render(
+    <TextTruncation text='This is some text to truncate' />,
+  );
+
+  const containerDiv = container.querySelector('div') as HTMLDivElement;
+  expect(containerDiv.style.display).toBe('flex');
```

Comment:
Are all those style checks really needed? Looks like a noise.

Reply:
Yeah, changed. Will need to write some proper tests now